data/nation/texan.ruleset

```diff
@@ -5,28 +5,35 @@ plural=_("?plural:Texans")
 groups="American"
 legend=_("Texas, \"the Lone Star State\" is the second largest and second\
  most populous state of the USA, famous for its \"larger than life\"\
- cowboy mentality. During its history, Texas has been ruled in full or in\
- part by six countries: France, Spain, Mexico, the Texas Republic, the\
- Confederacy and the USA - as expressed in the slogan \"six flags over\
- Texas\".")
+ cowboy culture and Spanish heritage. After the famous Battle of the Alamo de Bexar,\
+ The Texians defeated the Mexicans at San Jacinto and declared independence.\
```

Comment:
```suggestion
 the Texans defeated the Mexicans at San Jacinto and declared independence.\
```
("Texian" is archaic according to Wiki)

Reply:
It may be archaic, but it is an accurate representation of how the people referred to themselves at the time. I'll put inside quotes.